### PR TITLE
feat!: App SDK v4

### PR DESCRIPTION
**Pull request by shikaan**

### CODE REVIEW

File: dist/README.md [hunk 1]
```diff
@@ -1,11 +0,0 @@
-# !! WARNING !!
-
-`cf-extension.css` file in this directory is a prebuilt committed version of legacy
```
Comment (Jwhiles) [inline dist/README.md:3]: We're dropping this warning - but leaving the file? Is there a reason for that

Comment (andipaetzold) [reply dist/README.md:3]: We should not drop the files, but only the final sentence

> We can remove it as soon as we reach 4.0.

And update the unpkg url maybe

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## [3.34.3](https://github.com/contentful/ui-extensions-sdk/compare/v3.34.2...v3.34.3) (2021-05-18)
```
Comment (andipaetzold) [inline CHANGELOG.md:1]: The order of the changelog is wrong

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "contentful-ui-extensions-sdk",
   "description": "SDK to develop custom UI Extension for the Contentful Web App",
-  "version": "4.0.0-alpha.36",
+  "version": "3.34.3",
```
Comment (andipaetzold) [inline package.json:4]: Should probably be 3.42.0

File: .circleci/config.yml [hunk 4]
```diff
@@ -76,6 +76,7 @@ workflows:
             branches:
               only:
                 - master
+                - canary
```
Comment (Jwhiles) [inline .circleci/config.yml:79]: Do we still want this?

**Review by Jwhiles — APPROVED** [review]

**Pull request merged**